app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -305,6 +305,14 @@ export default Component.extend(ComposerUploadUppy, {
     }
   },
 
+  get _isNewTopic() {
```

Comment:
this most likely should be a computed property. e.g.

```suggestion
  @computed("composer.{creatingTopic,editingFirstPost,creatingSharedDraft}")
  get _isNewTopic() {
```